crates/router/src/connector/adyen/transformers.rs

```diff
@@ -220,7 +229,6 @@ pub struct AdyenResponse {
     refusal_reason: Option<String>,
     refusal_reason_code: Option<String>,
     additional_data: Option<AdditionalData>,
-    auth_code: Option<String>,
```

Comment:
Yes, this was added in AdyenResponse in the base PR; later, we decided to add a separate type for payouts response since some of the mandatory fields in payments response are not present in payouts response